tensorflow/python/platform/flags.py

```diff
@@ -34,11 +35,13 @@ class _FlagValues(object):
   def __init__(self):
     self.__dict__['__flags'] = {}
     self.__dict__['__parsed'] = False
+    self.__dict__['__required'] = []
```

Comment:
use self._required_flags maybe not work. use self._required_flags will call __getattr__ function which will throw an exception, because the _required_flags is not in __dict__['__flags'].